nat-handler.py

```diff
@@ -23,14 +23,17 @@
 
         {
             "eu-west-1a": {
+                "secondary_interface_id": "eni-abc123",
                 "route_table_id": "rtb-0e0ed06b",
                 "elastic_ip_allocation_id": "eipalloc-cc618fa9"
             },
             "eu-west-1b": {
+                "secondary_interface_id": "eni-def456",
                 "route_table_id": "rtb-090ed06c",
                 "elastic_ip_allocation_id": "eipalloc-c5618fa0",
             },
             "eu-west-1c": {
+                "secondary_interface_id": "eni-ghi789",
```

Comment:
Is it necessary to specify the secondary interface in the config? Couldn't it be derived at runtime? Or could we replace it with eg "eth1" to make it a bit less cryptic? (Not sure if the EC2 APIs allow any of this :)